app/modules/sightings/resources.py

```diff
@@ -344,7 +371,9 @@ def post(self, args):
 
         from app.modules.assets.schemas import DetailedAssetSchema
 
-        asset_schema = DetailedAssetSchema(only=('guid', 'filename', 'src'))
+        asset_schema = DetailedAssetSchema(
```

Comment:
Any reason this is subtly different to the other two and could the use of exclude= make this simpler?

Reply:
feel kinda like above; guess we may as well move all variations of schema as their own within schemas.py rather than sprinkled elsewhere in code with only/exclude variations.  so i will mess with this one as well.

Reply:
ok, i just did this modification with a single exclude. this should be good enough for this api response schema.